### PR TITLE
Add generate command

### PR DESCRIPTION
**Pull request by myokoym**
A part of #9

### CODE REVIEW

**Comment by myokoym** [comment]
We'll update templates when #16 is merged.

File: lib/daimon_skycrawlers/cli.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 require 'thor'
+require 'daimon_skycrawlers/generator/new'
 require 'daimon_skycrawlers/version'
 
 module DaimonSkycrawlers
   class CLI < Thor
+    register(Generator::New, "generate", "generate NAME", "Generate new project")
```
Comment (tricknotes) [inline lib/daimon_skycrawlers/cli.rb]: Is it a DSL, so is `()` unnecessary?
@okkez +1 / -1 ?

**Comment by okkez** [comment]
I like `new` than `generate`.
Because Rails uses `new` to create new project.
And I want to use `generate` to generate other snippets such as concrete crawlers and processors.

Like followings:

```
daimon-skycrawlers generate crawler MyCrawler
daimon-skycrawlers generate processor MyProcessor
daimon-skycrawlers generate migration CreateNewTable
```

File: lib/daimon_skycrawlers/cli.rb (hunk 1 above)
Comment (okkez) [inline lib/daimon_skycrawlers/cli.rb]: I want to use `()` here.
Because I want to break this line when this line becomes longer.

**Pull request merged**